patroni/utils.py

```diff
@@ -416,3 +418,27 @@ def cluster_as_json(cluster):
         if cluster.failover.candidate:
             ret['scheduled_switchover']['to'] = cluster.failover.candidate
     return ret
+
+
+def merge_paths(d1, d2):
```

Comment:
Let's give it a name `is_subpath(d1, d2)` and make it do just one thing and return `bool`.